be/src/storage/rowset/beta_rowset.cpp

```diff
@@ -243,6 +243,7 @@ Status BetaRowset::get_segment_iterators(const vectorized::Schema& schema, const
     seg_options.predicates_for_zone_map = options.predicates_for_zone_map;
     seg_options.use_page_cache = options.use_page_cache;
     seg_options.profile = options.profile;
+    seg_options.runtime_state = options.runtime_state;
```

Comment:
runtime_state belongs to query engine, I think we shouldn't pass it storage engine.